<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectProject</class>
 <widget class="QWidget" name="SelectProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Creator</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="newProjectButton">
       <property name="text">
        <string>New Project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="projectsList"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="loadProjectButton">
         <property name="text">
          <string>load Project</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selectFromDiskButton">
         <property name="text">
          <string>Select from disk</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
